express.js

```diff
@@ -39,22 +39,22 @@ app.use(middleware.clientChecks);
 
 //Set static public directory path
 app.use(express.static('public', {
-	immutable: true,
-	maxAge: 4 * 60 * 60 * 1000 // 4 hours
+    immutable: true,
+    maxAge: 4 * 60 * 60 * 1000 // 4 hours
 }));
 
 app.use(bodyParser.urlencoded({extended: false}));
 app.use(expressValidator({
-	customValidators: {
-		isEqual: function (value1, value2) {
-			return value1 === value2;
-		}
-	}
+    customValidators: {
+        isEqual: function (value1, value2) {
+            return value1 === value2;
+        }
+    }
 }));
 app.use(require('express-session')({
-	secret: process.env.SESSION_SECRET_KEY,
-	resave: false,
-	saveUninitialized: false,
+    secret: process.env.SESSION_SECRET_KEY,
+    resave: false,
```

Comment:
maybe revert auto formatter on this file (except for the one changed line)

Reply:
It's not auto formatter, it's me
All files normally have 4 spaces, the ones that don't appear weird in the github editor so I had to fix it